setup.py

```diff
@@ -59,7 +59,8 @@ def run_tests(self):
                         'more-itertools==5.0.0',
                         'tox==3.13.2',
                         'six==1.12.0',
-                        'PyYAML==5.3'],
+                        'PyYAML==5.3',
+                        'retrying==1.3.3'],
```

Comment:
Not necessarily an issue since this is a simple library, but just a heads up that this hasn't seen a release since 2014.

Reply:
I saw this too, but that should speak to how well the library works: There hasn't been a bug fix or feature addition in 6 years because of how well it *does* things already.

Reply:
I used a version of this library from the same author for Java (guava-retrying) and can attest that it works quite well for java - not necessarily any indication of the python implementation, of course, but some data point at least.